### ISSUE TITLE
 Doğrulanmış cevap barındıran soruları filtreleme 

### ISSUE THREAD

**Issue by turuncu122** [seahindeniz/BrainlyTools_Extension#295]
Content can now be filtered based on verified answers. In this way, verified content will be separated from other content.

**Comment by isaiasdesign03** [comment]
I agree that this would be very useful, as users are vindictively reporting questions answered by members of the community, such as moderators, who generally have their answers verified.
